src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -251,12 +251,12 @@ public double getInitialLod() {
             "(must be specified in increasing order)", optional = true)
     public List<Double> GVCFGQBands = new ArrayList<>(70);
     {
-        for (double i = -4.0; i <= 1; i = i + 0.5) {
+        for (double i = -2.5; i <= 1; i = i + 0.5) {
```

Comment:
Could you use `GVCFGQBands = Arrays.asList(MathUtils.createEvenlySpacedPoints(-2.5, 1, 8))`?